pages/dr9/catalogs.rst

```diff
@@ -34,7 +34,7 @@ such quantities are often quoted on the Vega system).
 ===================================== ============ ===================== ===============================================
 Name                                  Type         Units                 Description
 ===================================== ============ ===================== ===============================================
-``release``		              int16	 		         Unique integer denoting the camera and filter set used (`as documented here`_)
+``release``		              int16	 		         Integer denoting the camera and filter set used (`as documented here`_)
```

Comment:
I don't agree with this description.  I agree that it is not technically unique, but it does not denote the camera and filter set used.  If nothing else, it *also* encodes data reduction information in addition.  Let's come up with a better description.

Reply:
If you click through on the "as documented here" link for release, you'll be taken to this page:

https://github.com/legacysurvey/legacysurvey/blob/ADM-yet-more-DR9/pages/release.rst

which includes this language:
```
Note that all of the objects in DR9 across a total of 1691 bricks have a RELEASE number of 9012 instead of 9010. This was because these bricks required additional processing, which altered the sources that were extracted on these bricks. See also the known issues page.
```
I think it's pretty clear that "as documented here" ``RELEASE`` can encode processing information.

Reply:
However, that link is unavailable in Data Lab.  We really need the description to be self-contained.

Reply:
Is there a reason Data Lab can't contain a link to:

https://www.legacysurvey.org/release/

? I really don't want to write a paragraph of description in a one-line table.

Reply:
I'll try to add something short, if you like. But, it's really better to have the full link, so that you can click-through to the known issues page, which contains a file of all of the problematic bricks.

Reply:
OK, I've expanded the description slightly to:

```
Integer denoting the camera and filter set used, which will be unique for a given processing run of the data 
```

Will that work?

Reply:
That's fine, thank you.